integration_test/docker/celestia-bridge/lock

```diff
@@ -0,0 +1 @@
+1
```

Comment:
What is the point of committing a lock file into this repo?

Reply:
No point. Good catch!